README.md

```diff
@@ -25,3 +25,37 @@ Officially, *NAS2D* is supported on Windows (Vista+) and MacOS X (10.8+). Binari
 ## License
 
 NAS2D  is licensed under the zlib license. See LICENSE.txt for details.
+
+---
+
+## Vcpkg integration
+
+The NAS2D project uses Vcpkg to integrate any dependencies without explicitly adding the source to the project directly. If you do not want to install vcpkg in a user-wide context, vcpkg allows for a local NuGet package to be created. The following steps are specific to NAS2D:
```

Comment:
I feel like most people may want to just use a system wide install, and so instructions for that could be placed first. Perhaps also include instructions to remove the system wide install in case it causes problems for them.